src/ui/server/controllers/insurance/account/create/confirm-email/index.ts

```diff
@@ -14,16 +15,31 @@ export const PAGE_CONTENT_STRINGS = PAGES.INSURANCE.ACCOUNT.CREATE.CONFIRM_EMAIL
  * @param {Express.Response} Express response
  * @returns {Express.Response.render} Confirm email page
  */
-export const get = (req: Request, res: Response) => {
-  const { emailAddressToConfirm: email } = req.session;
-
-  delete req.session.emailAddressToConfirm;
-
-  return res.render(TEMPLATE, {
-    ...insuranceCorePageVariables({
-      PAGE_CONTENT_STRINGS,
-      BACK_LINK: req.headers.referer,
-    }),
-    email,
-  });
+export const get = async (req: Request, res: Response) => {
+  try {
+    const { accountIdToConfirm: exporterId } = req.session;
```

Comment:
if there is no id, should it still show the template?

If not, then should we not add a if (!exporterId) then go to problem with service

Reply:
It should still show the template - bit of a strange one. We temporarily store the ID in the session specifically for this page - then we remove it from the session before rendering the page. For security.

If a user refreshes this page, there will be no ID and therefore we can't display the email address. 

But we still want to say "sent you an email", just without the email address, instead of redirecting to an error page.